server/src/domain/storage-template/storage-template.service.ts

```diff
@@ -218,6 +218,7 @@ export class StorageTemplateService {
       ext,
       filetype: asset.type == AssetType.IMAGE ? 'IMG' : 'VID',
       filetypefull: asset.type == AssetType.IMAGE ? 'IMAGE' : 'VIDEO',
+      assetId: asset.id
```

Comment:
Can you also update the validate method in this same file to have a valid asset id?

Reply:
I think I did it, would you mind taking a look and lmk if this is what you meant?

Reply:
Looks good, thanks. That code runs when the migration is changed to validate the template. I'm not sure what happens if a referenced variable doesn't have any data. I assumed it would throw an error.